### ISSUE TITLE
reading rsk data incorrectly (?)

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#975]
I'm not completely sure this is a bug. I'm checking the dataset that yields `data(rsk)`. The screenshot shows what I get. In the ruskin software, I moved the mouse all the way to the left in the data trace, and hit the left-arrow a lot of times, to try to get the first data point. The numbers do not agree with what `read.rsk()` gets (see below). So that's Q1 (why are the numbers not the same). But the scarier thing is Q2 (why are the salinities so different).  I really don't think the very slight difference is giving this non-trivial salinity difference.

I wonder if Clark has ideas?

**SQLITE values**

```
$ sqlite3 060130_20150904_1159.rsk 
SQLite version 3.9.2 2015-11-02 18:31:45
Enter ".help" for usage hints.
sqlite> select * from data limit 1;
1441380732000|28.9279062632658|3.10050872892174|11.0638318035645
```

**R values**

```R
> rsk<-read.oce("060130_20150904_1159.rsk")
> ctd<-as.ctd(rsk)
> ctd[['conductivity']][1]
[1] 28.92790626327
> ctd[['temperature']][1]
[1] 3.100508728922
> rsk[['pressure']][1]
[1] 11.06383180356
> ctd[['pressure']][1]
[1] 0.9313318035645
> ctd[['salinity']][1]
[1] 31.5709065603
```

**Ruskin values**

<img width="607" alt="screen shot 2016-05-14 at 5 57 19 pm" src="https://cloud.githubusercontent.com/assets/99469/15270675/41b2eea4-19fe-11e6-9eb6-912ccbbeae7a.png">

**Comment by dankelley** [comment]
To sum up the previous: sea pressure is very slight different in ruskin, while C and T are identical to the digits displayed by ruskin. But S differs by 0.005PSU, which is a lot.

Any ideas on this, Clark? Either oce or Ruskin is quite a lot more wrong than I'd expect.  (PS. my local default is the UNESCO eqn of state, so that's not it.)

**Comment by dankelley** [comment]
Sorry, I see the result. Will fix after my run.

**Comment by dankelley** [comment]
Fixed in `develop` commit 38e8aac53064665c869ed1d7f67eb89237f36c3f

The problem was that I was computing S from the p in the rsk, not the ctd. Now the S matches up.  I still don't see why the "sea pressure" is out by a tiny amount, but suspect it's something with rounding error.

I added some tests (not exactly on these lines) at the bottom of `tests/testthat/test_ctd.R`.

Closing.
